Source/carthage/Archive.swift

```diff
@@ -103,6 +103,8 @@ public struct ArchiveCommand: CommandType {
 						.map { ($0 as NSString).lastPathComponent }
 						.filter { $0.hasSuffix(".framework") }
 
+					let relativePaths = paths.map { $0.stringByReplacingOccurrencesOfString(options.directoryPath, withString: ".") }
```

Comment:
Yup! Or we could pass both the relative and absolute paths together in a tuple. That might be clearest.